File: CursoPOO/PHP/car.php
<?php
require_once('account.php');

class Car {
    public $id;
    public $license;
    public $driver;
    public $passenger;


    public function __construct($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
    }

   
}
?>

File: CursoPOO/Java/Car.java
package CursoPOO.Java;

public class Car {
    int id;
    String license;
    Account driver;
    int passenger;

    public Car(String license, Account driver){  //metodo constructir
        this.license = license;
        this.driver = driver;
    }

    void printDataCar(){
        System.out.println("License: " + license + ", Name Driver: "+ driver.name);
    }
}
